compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/SymbolResolver.java

```diff
@@ -1322,4 +1324,48 @@ private boolean isMemberAllowed(SymbolEnv env, BSymbol symbol) {
                 && env.enclInvokable.symbol.receiverSymbol.type.tsymbol == symbol.owner
                 || isMemberAllowed(env.enclEnv, symbol));
     }
+
+    public void loadFunctionalConstructors() {
+        BPackageSymbol xmlModuleSymbol = symTable.langXmlModuleSymbol;
+        if (xmlModuleSymbol == null) {
+            return;
```

Comment:
Is this a valid case, the XML module symbol being `null`? Shouldn't the XML lang lib always be available by this point? In which case, this is an invalid state

Reply:
This[[1]](https://scans.gradle.com/s/c5uvl2bsschds) is without this exit condition, couldn't figureout why it fails like this 
[1] - https://scans.gradle.com/s/c5uvl2bsschds

Reply:
Hmm.. Could it be a Gradle dependency issue? I had a look at the log. `createBStringBalo` task doesn't seem to have run for the `ballerina-lang:xml` module. Could it be that?